extension/popup/popupView.jsx

```diff
@@ -589,23 +595,50 @@ const SearchResultsContent = ({
     onNextSearchResultClick();
   };
 
+  const searchCard = () => (
+    <button class="invisible-button" onClick={onSearchCardClick}>
+      <img id="search-image" alt={imgAlt} style={cardStyles} src={card.src} />
+    </button>
+  );
+
+  const qaCard = () => (
```

Comment:
I don't understand the name `qaCard`? Maybe `AnswerCard` would be the better name?

Reply:
It was referred to as the Q&A card in issue #1267 so I named it `qaCard`, haha. `AnswerCard` is definitely a better name, I will make this change.